aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -497,11 +497,31 @@ See the [Binding Failures](#bf) section for more information.
 
 The following types are bound without explicit attributes:
 
-* `HttpContext` : The context which holds all the information about the current HTTP request or response.
-* `HttpRequest` : The HTTP request
-* `HttpResponse` : The HTTP response
-* <xref:System.Threading.CancellationToken> : The cancellation token associated with the current http request.
-* <xref:System.Security.Claims.ClaimsPrincipal> :  The user associated with the request (`HttpContext.User`).
+* <xref:Microsoft.AspNetCore.Http.HttpContext>: The context which holds all the information about the current HTTP request or response:
+
+  ```csharp
+  app.MapPost("/", context => context.Response.WriteAsync("Hello World"));
```

Comment:
Is it worth changing this to `(HttpContext context)` to make it easier for those that need to add other parameters to the signature?

Reply:
OK lets add the type `HttpContext`.